src/custom/theme/baseTheme.tsx

```diff
@@ -444,9 +444,9 @@ export const ThemedGlobalStyle = createGlobalStyle`
   }
 
   // START - Modal overrides
-  ${HeaderText} {
+  /* ${HeaderText} {
```

Comment:
Do we need this in the future?
If yes, then would be great to add a comment, otherwise, just delete it